packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1235,6 +1235,16 @@ export class TransactionController extends BaseController<
     return this.nonceTracker.getNonceLock(address);
   }
 
+  /**
+   * Filters out the unapproved transactions from state
+   */
+  clearUnapprovedTxs() {
```

Comment:
Shall we use the full form so `clearUnapprovedTransactions` to match all our types which are now using `Transaction`?

Reply:
Sure, I didn't want to change name of it but make sense